packages/dids/src/methods/did-dht.ts

```diff
@@ -1026,6 +1036,12 @@ export class DidDhtDocument {
           // Convert the public key from a byte array to JWK format.
           let publicKey = await DidDhtUtils.keyConverter(namedCurve).bytesToPublicKey({ publicKeyBytes });
 
+          // DID DHT spec requires `alg` in keys in the DID document
+          publicKey.alg = parsedAlg || getJoseSignatureAlgorithmFromPublicKey(publicKey);
```

Comment:
`getJoseSignatureAlgorithmFromPublicKey` seems to return `EdDSA` when `crv` is `Ed25519`.

Maybe what need to be done is to extend the vector tests such that:

DID document -> DNS records -> DID document ends up being the same. Right now the vector tests stops at DNS records conversion.